recipes/taskipy/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "taskipy" %}
+{% set version = "1.10.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0006429f708f530fc7add28ca51fd41f6c6e1fbe86763ff05125e1af3e8bdc7e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - task = taskipy.cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - poetry
+  run:
+    - python >=3.7
+    - pip
+    - poetry
+    - colorama >=0.4.4
+    - psutil >=5.7.2
```

This will ensure the package fails during the solve and not the pip test. Otherwise it will be installable on Windows! Also, this way this package is ready in case someone submits mslex to conda-forge.


```suggestion
    - __unix  # [unix]
    - __win   # [win]
    - mslex >=0.3.0  # [win]
    - psutil >=5.7.2
```

---

Done. Thanks again!